lib/src/api/login_api_service.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class APIService {
+  Future<Map> login(String userId, String password) async {
+    dynamic postBody = <String, String>{"userId": userId, "password": password};
+    String url = "http://52.136.112.220:57842/api/auth/login";
+    var headers = {'Content-Type': 'application/json'};
+    SharedPreferences prefs = await SharedPreferences.getInstance();
```

Comment:
Save the `access_token` into `FlutterSecureStorage`. And save `is_logged_in` = true in `SharedPrefs`.